owner/src/main/java/org/aeonbits/owner/Converters.java

```diff
@@ -144,8 +138,29 @@ Object tryConvert(Method targetMethod, Class<?> targetType, String text) {
         }
     },
 
-    PROPERTY_EDITOR {
+    CLASS_IMPLEMENTING_CONFIG_INTERFACE {
+        @Override
+        Object tryConvert(Method targetMethod, Class<?> targetType, String text) {
+            if (!Reflection.getAllInterfaces(targetType).contains(Config.class)) return SKIP;
+            try {
+                Map<String, String> imports = new HashMap<String, String>();
+                String namespace = ConfigFactory.getProperty("nested.config.prefix");
+                if (namespace == null) {
+                    namespace = "ns";
+                }
+                imports.put(namespace, text);
+                Object result = ConfigFactory.create((Class<? extends Config>) targetType, imports);
```

Comment:
The problem here, is that every time this call is done, a new object (a dynamic proxy) is created; but that's also consistent with the rest of the code. Maybe we can add a class level annotation that allows owner to use a cached instance (see org.aeonbits.owner.ConfigCache.java, http://owner.aeonbits.org/docs/singleton/)